docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -308,10 +343,13 @@ agent you want to unenroll.
 [[ca-cert-testing]]
 == On {fleet-server} startup, ERROR seen with `State changed to CRASHED: exited with code: 1`
 
+//QUESTION: Is this section still true? It does appear that we get a better error now,
+//plus it seems that users are overlooking this section. Should I keep this section or remove it?
```

Comment:
What makes you think they are overlooking it and why? If that is the only error its not useful without more information, so having this could still be helpful. I'm not sure if its still true or not.

Reply:
OK, I just wasn't sure if this problem would still result in a crash. Sounds like it does, so I will keep it. I think users who are looking for the certs-related error do not find this section, but it's OK to cover the details in both. I will leave as-is.